virttest/utils_sys.py

```diff
@@ -72,3 +72,26 @@ def get_host_bridge_id(session=None):
 
     host_bridges = re.findall(hostbridge_regex, output)
     return host_bridges if host_bridges else []
+
+
+def get_pids_for(process_names, session=None):
+    """
+
+    Given a list of names, retrieve the
+    PIDs for matching processes
+    Sort of equivalent to: 'ps aux | grep name'
+    :param process_names: List of process names to look for
```

Comment:
Please add one more blank line above this line which is to apply to docstring style. Thanks.
And also add a docstring for return value.